### DISCUSSION TITLE
[A-L] (2023/24) Lezione 09/10/2023 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#164] [Esercizi - Viaggi/Piazza]
**Esercizio 1 (Unicità dell'Inverso):** Sia $(G,\star)$ un gruppo, verificare che dato $g \in G$, l’elemento $g'$ di cui in **(3)** è unico. Esso è detto inverso di $g$ ed è denotato $g^{−1}$.
**(3)** $\forall g \in G,\exists g'\in G : g\star g'=e=g'\star g$
> Suggerimento: assumete che ce ne siano due e dimostrate che devono essere uguali.

**Comment by daromacarlo** [comment]
vero, sorry, sarà l'orario... 😂
![Immagine 2023-10-11 104816](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/bd3f3565-bd66-45c1-a5c6-8ba01ac43a12)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/6867e42f-1e03-493f-b8d4-5ebe070716e9)
